src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENT_PRICE = 40;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const FULL_TERM = 7;
+  const FULL_TERM_DISCOUNT = 50;
+
+  const DISCOUNT =
+    days >= FULL_TERM
+      ? FULL_TERM_DISCOUNT
+      : days >= SHORT_TERM
+        ? SHORT_TERM_DISCOUNT
+        : 0;
```

Comment:
Avoid nested ternary operators, they are difficult to read.
In this case, it's better to calculate base price first and then in different conditions return different values(with discount or not), something like this
![image](https://github.com/user-attachments/assets/8503a8ab-d06b-4fc2-bbc0-7910be46b04d)